connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -45,6 +46,11 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
   private val executions: ConcurrentMap[String, ExecuteHolder] =
     new ConcurrentHashMap[String, ExecuteHolder]()
 
+  val errorIdToError =
```

Comment:
doc? what does it collect? Is it root errors or any error? What is the relationship between errors in the cache?